src/miniscript/mod.rs

```diff
@@ -909,4 +909,24 @@ mod tests {
             .to_string()
             .contains("unprintable character"));
     }
+
+    #[test]
+    fn temp() {
```

Comment:
Can you rename this test? Otherwise ack cec1e08351d97cb5acb56fc3b8bfc505c5683fac

Reply:
Oops yeah, this was for initial testing.